src/main/java/com/jcabi/xml/ClasspathInput.java

```diff
@@ -121,10 +121,12 @@ public String getEncoding() {
     @Override
     public String getStringData() {
         try {
-            return IOUtils.toString(
+            final String data = IOUtils.toString(
                 this.stream,
                 Charset.forName("UTF-8")
             );
+            this.stream.close();
```

Comment:
@amihaiemil OK, the only client I see is `ClasspathResolver` - it should be responsible for  closing the stream, not the `ClasspathInput`